csrc/device_lower/lower2device.h

```diff
@@ -230,6 +243,9 @@ class GpuLower : public NonCopyable {
   // Lowered Kernel IR
   std::unique_ptr<kir::Kernel> kernel_;
 
+  // Passes to lower kernel
```

Comment:
Please also mention they are executed in this order

Reply:
added